tests/integration/conftest.py

```diff
@@ -147,3 +147,22 @@ def deactivated_scm_service(scm_service):
         scm_service.save()
 
     return scm_service
+
+
+@pytest.fixture
+def scm_repository(scm_service, credential):
```

Comment:
The tests are okay with defining this fixture after the application fixture?

Reply:
I just moved down to make it more readable.